linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -143,16 +140,14 @@ static void threadpool_dispose(THREADPOOL* threadpool)
 
 `threadpool_dispose` frees the resouces associated with threadpool.
 
-**SRS_THREADPOOL_LINUX_07_013: [** `threadpool_dispose` shall perform an implicit close if `threadpool` is open. **]**
+**SRS_THREADPOOL_LINUX_07_089: [** `threadpool_dispose` shall signal all threads that the threadpool is closing by setting the flag to stop the threads to 1. **]**
```

Comment:
>closing [](http://example.com/codeflow?start=104&length=7)

If we say "shall signal all threads to return" we don't have to mention closing (since we do not have close as a state any longer)

Reply:
Thanks. Was unable to find right word to replace threadpool closing. Will do this.

Reply:
stopping would work here.